packages/node/src/events/propose-install-virtual/operation.ts

```diff
@@ -1,8 +1,8 @@
 import { Node } from "@counterfactual/types";
 
+import { getOrCreateVirtualChannel } from "../../methods/app-instance/propose-install-virtual/operation";
```

Comment:
Perhaps lift to util file

Reply:
If other contexts end up needing this we can do that, but this is relatively specific to the context of virtual channels so might be better to leave it here (as opposed to a more general util location)